Nodejs/Product/Nodejs/Resources.resx

```diff
@@ -600,4 +600,13 @@ You will need to restart Visual Studio after installation.</value>
   <data name="SurroundWith" xml:space="preserve">
     <value>Surround With</value>
   </data>
+  <data name="TsdErrorOccured" xml:space="preserve">
+    <value>Error installing typings.</value>
+  </data>
+  <data name="TsdInstallCompleted" xml:space="preserve">
+    <value>Successfully installed typings.</value>
+  </data>
+  <data name="TsdNotInstalledError" xml:space="preserve">
+    <value>Could not find TSD. Make sure you have installed it globally.</value>
```

Comment:
I agree that we should either: install Tsd automatically if it is not found, or, as you state, tell that user what steps they should take. I think we are targeting the second option for now.

For errorDescriptionAndResolution, how about `Could not find TSD TypeScript package manager. Please install it globally by running 'npm install -g tsd'`.

I think that clearly says what is wrong and how to fix it. We may also want to include a link to a help page or something.